README.md

```diff
@@ -242,6 +248,29 @@ run_list(
 
 For Windows, because the parameter `oneagent_download_dir` is a string variable, 2 backslashes are required within the file path. Since the OneAgent install parameter `INSTALL_PATH` can be defined within the `oneagent_install_params_hash` hash map, no escaping is needed.
 
+#### OneAgent installation using a PaaS token from encrypted data bag
+
+Requires creating encrypted data bag:
+
+```
+cat data_bags/vault/dynatrace.json
+{
+  "id": "dynatrace",
+  "paas_token": "dt0c01.ST2EY72KQINMH574WMNVI7YN.G3DFPBEJYMODIDAEX454M7YWBUVEFOWKPRVMWFASS64NFH52PX6BNDVFFM572RZM"
```

Comment:
please remove the token here and make sure it has been invalidated.

Reply:
I took this token from the official Dynatrace documentation, so it's not related to any production deployment, I believe.

Reply:
Removed the token anyway. Sorry for the slow reply.